recipes/miniwdl/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "miniwdl" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fbb646a09af08f0d13f2f197eba01db576a70745190fa8546d50d165f21682d4"
+
+build:
+  skip: true  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - pbr
+    - python>3.6.0
+    - setuptools
+    - docker-py
+    - lark-parser==0.7.1
+
+  run:
+    - python>3.6.0
+    - setuptools
+    - docker-py
+    - lark-parser==0.7.1
```

Comment:
This also needs a space like after the name (or to be formatted like Marcelo suggested)